### DISCUSSION TITLE
Is Nextcloud AIO made to stay?

### DISCUSSION THREAD

**Discussion by Stefomat** [nextcloud/all-in-one#1991] [Questions] [answered]
Asking this question is always uncomfortable, no matter what the project is.
Nextcloud AIO is currently enjoying great popularity. Many users, lots of stars on GitHub, development is in full swing. Nextcloud AIO is even mentioned on the Nextcloud download page directly under Enterprise Support and before the .tar.gz download.

I am currently working on getting Nextcloud AIO up and running in my company. Not a large environment, not many users, no need for enterprise support. But, I don't want to be in the situation of having to switch to another Nextcloud installation in a year and a half. Considering that AIO itself is only a little over a year old.
So, can you maybe share some information for which users Nextcloud AIO is primarily developed for?
Asked bluntly, is AIO a side project of some Nextcloud developers for the Homelab or can it also be used in a company in good conscience?

I don't want to degrade anyone's work with this question, I don't want to insult anyone! It's just the thoughts that go through my head when I choose software and the type of installation for my firm.

Thanks

**Comment by szaimen** [comment, marked as answer]
> Is Nextcloud AIO made to stay?

Yes, it is
